Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -23,15 +23,9 @@ namespace ExpansionInterface
 // Multiple parts of this implementation depend on Dolphin
 // being compiled for a little endian host.
 
-CEXIETHERNET::CEXIETHERNET()
+CEXIETHERNET::CEXIETHERNET(BBADeviceType type)
 {
-  tx_fifo = std::make_unique<u8[]>(BBA_TXFIFO_SIZE);
-  mBbaMem = std::make_unique<u8[]>(BBA_MEM_SIZE);
-  mRecvBuffer = std::make_unique<u8[]>(BBA_RECV_SIZE);
-
-  MXHardReset();
-
-  // Parse MAC address from config, and generate a new one if it doesn't
+  // Parse BBA (TAP) MAC address from config, and generate a new one if it doesn't
```

Comment:
Same reasoning about TAP vs. XLink Kai for the MAC as in other places; it isn't TAP specific as far as I can see.